external-videos.php

```diff
@@ -259,26 +255,25 @@ function admin_scripts( $hook ) {
   static function get_options(){
 
     // Load existing plugin options
-    $raw_options = get_option( 'sp_external_videos_options' );
+    $options = get_option( 'sp_external_videos_options' );
     // echo '<pre>$raw_options: '; print_r($raw_options); echo '</pre>';
 
+    if( !$options ) {
+      $options = array();
+    }
+    // Upgrade options to v1.0: move author options to hosts
+    $options = SP_External_Videos::convert_author_options( $options );
+
     // Set defaults for the basic options
-    if( !$raw_options ) {
-      $options = array( 'version' => 1, 'rss' => false, 'delete' => true, 'hosts' => array(), 'attrib' => false, 'loop' => false, 'slug' => 'external-videos');
```

Comment:
You should not remove this - this is initialization for the first time somebody installs the plugin!